client/app/license/order/license-order.html

```diff
@@ -217,19 +217,14 @@ <h2 data-translate="license_order_step4_title"></h2>
                                     data-ng-if="agoraEnabled"
                                     data-translate="license_order_step4_contracts_agora_ask">
                             </strong>
-                            <table class="table table-hover">
-                                <tbody>
-                                    <tr data-ng-repeat="contract in durations.details[selected.duration].contracts">
-                                        <td>
-                                            <a target="_blank"
-                                               rel="noopener"
-                                               data-ng-href="{{ contract.url }}"
-                                               data-ng-bind="contract.url">
-                                            </a>
-                                        </td>
-                                    </tr>
-                                </tbody>
-                            </table>
+                            <span data-ng-repeat="contract in durations.details[selected.duration].contracts">
+                                <a target="_blank"
+                                   rel="noopener"
+                                   class="d-inline-block"
+                                   data-ng-href="{{ contract.url }}"
+                                   data-ng-bind="contract.url">
```

Comment:
Please add:
* an icon to tell users you will open a new tab
* a text for screen-readers that you will open a new tab (the translation id should already exist and be something like `common_new_tab`)